nucypher/config/node.py

```diff
@@ -362,7 +363,7 @@ def from_configuration_file(cls, filepath: str = None, **overrides) -> 'NodeConf
                                                   deserializer=cls.NODE_DESERIALIZER)
 
         payload.update(dict(node_storage=node_storage))
-        return cls(is_me=True, **{**payload, **overrides})
+        return cls(**{**payload, **overrides})
```

Comment:
Can we name this or something?  It looks like some kind of new age emoji.

Reply:
I admit the sequence `(**{**` is a bit emoji-like

Reply:
Named the call to `__init__` - :heavy_check_mark: